docs/versions.md

```diff
@@ -21,26 +21,34 @@ server = TSC.Server('http://SERVER_URL')
 print(server.version)
 ```
 
-For example, the code might display version `2.3`.
+For example, the code might display version `2.3`. 
 
-## Use another version of the REST API
+## Use the REST API version supported by the server
 
-To use another version of the REST API, set the version like so:
+To always use the latest version of the REST API that is supported by the instance of Tableau Server you are connecting to, set the version using the `use_server_version()` method:
 
 ```py
 import tableauserverclient as TSC
 
 server = TSC.Server('http://SERVER_URL')
```

Comment:
probably have it be `server = TSC.Server('http://SERVER_URL', use_server_version=True)` since that's more compact

Reply:
Yes. That is pretty slick.